examples/animation/arms/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <title>Arms</title>
     <meta name="description" content="Arms - A-Frame">
-    <script src="../../../dist/aframe.js"></script>
+    <script src="../../../dist/aframe-dev.js"></script>
```

Comment:
Example builds are rewritten by the dev server to point to current tree. No need to change.